tests/src/test/scala/whisk/core/controller/test/RulesApiTests.scala

```diff
@@ -563,7 +563,7 @@ class RulesApiTests extends ControllerTestCommon with WhiskRulesApi {
         WhiskRuleResponse(
           namespace,
           rule.name,
-          Status.ACTIVE,
+          Status.INACTIVE,
```

Comment:
Status is set to `INACTIVE` in this file because the trigger route is not accessible in the unit test, resulting in an `INACTIVE` status when a rule is updated.